main.go

```diff
@@ -469,6 +469,11 @@ func main() {
 		*s3prefix = *logURL
 	}
 
+	_, err := url.ParseRequestURI(*s3bucket)
```

Comment:
Looking at the documentation for [ParseRequestURI](https://pkg.go.dev/net/url#ParseRequestURI) it doesn't seem to say it will reject absolute URIs (starting with a scheme). It actually seems to say the opposite - that it will accept them. Does this definitely work?

Reply:
From my manual testing, yes it works.
```
$ go run . -log-url https://oak.ct.letsencrypt.org/2023 -tile-size 256 -s3-bucket s3://whatever
2023/09/11 14:36:11 scheme provided for s3-bucket, but should not be
exit status 1

$ go run . -log-url https://oak.ct.letsencrypt.org/2023 -tile-size 256 -s3-bucket noscheme
^Csignal: interrupt



```